### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

- Closes #580
- Closes #579
- Closes #578 
- Closes #577
- Closes #576
- Closes #575

### CODE REVIEW

**Comment by tdeekens** [comment]
Warming up on a Monday here.

File: application-templates/starter/package.json [hunk 1]
```diff
@@ -33,6 +33,6 @@
     "@commercetools-frontend/mc-scripts": "13.3.5",
     "enzyme": "3.9.0",
     "jest": "24.7.1",
-    "react-testing-library": "6.1.2"
+    "react-testing-library": "7.0.0"
```
Comment (tdeekens) [inline application-templates/starter/package.json]: as mentioned: selectors now throw if the find more than one element.

File: package.json [hunk 2]
```diff
@@ -62,22 +62,22 @@
...
     "eslint-plugin-cypress": "2.2.1",
     "eslint-plugin-import": "2.17.2",
-    "eslint-plugin-jest": "22.4.1",
+    "eslint-plugin-jest": "22.5.1",
     "eslint-plugin-jsx-a11y": "6.2.1",
     "eslint-plugin-prefer-object-spread": "1.2.1",
     "eslint-plugin-prettier": "3.0.1",
     "eslint-plugin-react": "7.12.4",
-    "husky": "1.3.1",
+    "husky": "2.1.0",
```
Comment (tdeekens) [inline package.json]: Dropped node@6 support.

File: packages/mc-scripts/package.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
     "graphql-tag": "2.10.1",
     "html-webpack-plugin": "3.2.0",
     "json-loader": "0.5.7",
-    "mini-css-extract-plugin": "0.5.0",
+    "mini-css-extract-plugin": "0.6.0",
```
Comment (tdeekens) [inline packages/mc-scripts/package.json]: Actually now supports HMR. Not that we need it.

File: package.json [hunk 4]
```diff
@@ -51,7 +51,7 @@
     "website"
   ],
   "dependencies": {
-    "gatsby-cli": "^2.5.9"
+    "gatsby-cli": "^2.5.12"
```
Comment (emmenko) [inline package.json]: Oh wait, this should go to the website package.

File: package.json [hunk 5]
```diff
@@ -86,11 +86,11 @@
     "jest-runner-stylelint": "1.0.0",
     "jest-silent-reporter": "0.1.2",
     "jest-watch-typeahead": "0.3.0",
-    "lerna": "3.13.3",
+    "lerna": "3.13.4",
```
Comment (emmenko) [inline package.json]: Update `lerna.json` as well?

File: packages/application-components/package.json [hunk 6]
```diff
@@ -45,21 +45,21 @@
...
+    "@storybook/react": "5.0.11",
+    "@storybook/theming": "5.0.11",
     "html-loader": "0.5.5",
     "markdown-loader": "5.0.0",
     "moment-locales-webpack-plugin": "1.0.7",
     "react": "16.8.6",
     "react-dom": "16.8.6",
     "react-intl": "2.8.0",
-    "react-testing-library": "6.1.2",
+    "react-testing-library": "7.0.0",
```
Comment (emmenko) [inline packages/application-components/package.json]: Please update the peer dependency range as well

File: package.json (hunk 4 above)
Comment (tdeekens) [reply package.json]: 41f9b1c

File: package.json (hunk 5 above)
Comment (tdeekens) [reply package.json]: 9b3f606

File: packages/application-components/package.json (hunk 6 above)
Comment (tdeekens) [reply packages/application-components/package.json]: d0a3a5f. We can support both. It's more of a consumer decision I would say as we support the stricter v7 interally.

**Pull request merged**